presets/90-systemd.preset

```diff
@@ -20,6 +20,7 @@ enable getty@.service
 enable systemd-boot-update.service
 enable systemd-confext.service
 enable systemd-homed.service
+enable systemd-homed-*.service
```

Comment:
uh, we know those names. Let's spell them out please. don't use wildcards here.

I think using wildcards where there's no good reason to me is not a good idea, it makes things just hard to read, because people won't know what precisely is meant with this unless they look at what is actually installed on their systems.

Reply:
I used a wildcard because a) clearly we're not very good at remembering to list new units here, b) this is under our own namespace so the wildcard wouldn't match something accidentally. But it's easy enough to switch to an explicit list, wil do that.